mavros/launch/apm_config.yaml

```diff
@@ -122,7 +122,9 @@ setpoint_velocity:
 # waypoint
 mission:
   pull_after_gcs: true  # update mission if gcs updates
-
+  use_mission_item_int: true # use the MISSION_ITEM_INT message instead of MISSION_ITEM
+                             # for uploading waypoints to FCU
```

Comment:
if it is set to true but the flight controller still requests just a MISSION_ITEM like ardupilot then it will try using INT. This flag is more for explicitly turning it off, I'm not sure if it's necessary but it did make confirming that MISSION_ITEM was still functioning as intended easier.